docs/1.5/extensions/footnotes.md

```diff
@@ -2,12 +2,23 @@
 layout: default
 title: Footnote Extension
 description: The FootnoteExtension adds the ability to create footnotes in Markdown documents.
+redirect_from: /extensions/footnotes/
```

Comment:
I wasn't sure about this, but it appeared to be missing.

Reply:
Yep this is perfect!